### ISSUE TITLE
Issues in font rendering

### ISSUE THREAD

**Issue by Doc73** [nickjwhite/diogenes#7]
I'm testing your app and I found some errors in font rendering, maybe related to CSS.
Please, see at following screenshots of LSJ s. v. λαμνβάνω, the firt taken from original Diogenes, the second from yours.
You'll see these errors:
1. missing spaces between words;
2. some wrong character.

Many thanks in advance,
Domenico

**Comment by Doc73** [comment]
Old Diogenes

![old](https://user-images.githubusercontent.com/1306260/44520245-a87cea80-a6cf-11e8-8dcf-27027ce50bf4.png)

**Comment by Doc73** [comment]
New Diogenes

![new](https://user-images.githubusercontent.com/1306260/44520285-bfbbd800-a6cf-11e8-8860-8e163985b96c.png)

**Comment by JayBrown** [comment]
Glad to see I'm not alone. ;)

**Comment by Doc73** [comment]
@JayBrown 
Ah! My bug report seems a duplicate of your #6

**Comment by Doc73** [comment]
Found a workaround!!!!
Edit file `.../Diogenes/diogenes-nwjs/diogenes-browser/perl/diogenes.css` and add
```
a::after {
	content: " ";
}
a::before {
	content: " ";
}
```

after 

```
a:hover {
  text-decoration: underline;
}
```

**Comment by nickjwhite** [comment]
Thanks a lot for pointing this out and suggesting a fix. I fixed it with c62062d. I'm pretty sure it's correct and robust, but a bit of testing would be very much appreciated if you can spare the time!

**Comment by Doc73** [comment]
Thank you very much for updating your app! :+1:
